### ISSUE TITLE
Display alert/advisory on Recreation Site and Trail page

### ISSUE THREAD

**Issue by BAAlexK** [bcgov/nr-rec-resources#158]
#### Description:

**As a** *(User Type/Persona)* **I want** *(Feature/enhancement)* **So That** *(Value, why is this wanted, what is the user trying to accomplish)*

[Description of the ticket]

#### Acceptance Criteria:
- [ ] Display date of when it was posted/updated and next expected update
- [ ] Show description of alert including potential concerns around access - (busy industrial roads)
- [ ] Clearly indicate if visitor needs to take a specific action of because of an event (flood, fire, landslide?)
- [ ] shown on page to make it obvious
- [ ] ...
- [ ] ...

#### Development Checklist:
- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies
- Blocked by
- Blocking

#### Notes:
- For MVP use existing closure and alert data/info
- Should include a Alerts/Advisory banner on home page? quick direct to critical alerts and information in an area?
-

**Comment by airliah** [comment]
this is not happening for some months, I'm going to close for now

**Comment by BAAlexK** [comment]
@airliah it doesnt need to be closed, tickets can sit in the backlog until either they are worked on or deleted/removed (won't do)
